src/lxc/lxccontainer.c

```diff
@@ -5725,3 +5725,23 @@ bool lxc_has_api_extension(const char *extension)
 
 	return false;
 }
+
+int seccomp_notify(struct lxc_container *c, unsigned int cmd, int fd)
```

Comment:
don't you need to initialize this in lxc_container_new() too?

Reply:
hahah, yes :D

Reply:
done